### PR TITLE
Feature/vmware nova proxy

### PR DESCRIPTION
**Pull request by csymons-suse**

### CODE REVIEW

File: xml/install-esx_computes-ovsvapp.xml [hunk 1]
```diff
@@ -53,6 +53,55 @@
...
+   <literal>nova-compute</literal> VM is the OVSvApp Nova VCDriver that talks
+   to the VMware vCenter server to perform VM operations such as VM creation
+   and deletion.
+  </para>
+ </section>
+ <section>
+  <title>VM Appliances Used in OVSvApp Implementation</title>
+  <para>
+   The default configuration deployed with the &clm; for VMware ESX hosts uses
+   service appliances that run as VMs on the VMware hypervisor.  There is one
```
Comment (asettle) [inline xml/install-esx_computes-ovsvapp.xml]:
```suggestion
service appliances that run as VMs on the VMware hypervisor. There is one
```

**Comment by asettle** [comment]
Also @dmpop is away at the moment so feel free to merge when you're ready.

**Comment by csymons-suse** [comment]
> This looks good to me. I wasn't sure about the image changes they look identical to me.
@tbosworth7 
I didn't make changes to any images; nothing in your email about this that I can see. What did you expect to see? What images should be changed?

**Comment by tbosworth7** [comment]
I saw images in the pull request.  I expected no images to be updated.

**Comment by csymons-suse** [comment]
> I saw images in the pull request. I expected no images to be updated.

ah yeah...those images reported unoptimized when I ran the trial HTML render. I optimized them in a separate commit. This has nothing to do with nova-proxy, but it _does_ have to do with efficient rendering.

**Pull request merged**